iptables/iptables.go

```diff
@@ -220,6 +220,16 @@ func (ipt *IPTables) DeleteIfExists(table, chain string, rulespec ...string) err
 	return err
 }
 
+// List rules in specified table/chain
+func (ipt *IPTables) ListById(table, chain string, Id int) (string, error) {
```

Comment:
This looks good. One minor nit: `Id` should be lower-cased. Once that's fixed we can merge :-)

Reply:
Done